### DISCUSSION TITLE
How can I get user with by a phone number?

### DISCUSSION THREAD

**Discussion by Sipioteo** [zitadel/zitadel#6904] [Q&A]
Hey all, I'm building a "telegram like" onboarding and I want user to login and access the service only through phone number. How can I filter (or search) the user by a given phone number? (The management api has only the email query, not the phone query)

Thank you all

**Comment by mffap** [comment]
at this time this query is in fact not supported in https://zitadel.com/docs/apis/resources/mgmt/management-service-list-users

If this would be useful to you, please open an issue as improvement idea and we will look into it.
